lib/slack.rb

```diff
@@ -15,7 +15,7 @@ def post_confused_features
 
       message = confused_features.map { |f| "- '#{f.name}'" }.join("\n")
 
-      post(text: "🥴 Uh-oh! The following feature flags are enabled on production, but not on all other environments:\n\n#{message}\n\n<https://apply-ops-dashboard.herokuapp.com/features|:shipitbeaver: Check the feature flags dashboard>")
```

Comment:
But 🥴 is the official environmental confusion emoji on the dashboard!

Reply:
🥴

Reply:
🥴